internal/pipe/git/git.go

```diff
@@ -32,6 +34,10 @@ func (Pipe) Run(ctx *context.Context) error {
 	ctx.Git = info
 	log.Infof("releasing %s, commit %s", info.CurrentTag, info.Commit)
 	ctx.Version = strings.TrimPrefix(ctx.Git.CurrentTag, "v")
+	if ctx.Config.Deterministic {
+		ctx.Date = info.CommitDate
```

Comment:
maybe add a new template field `.CommitDate` instead? Then the user can use it where they choose

Reply:
Done